VSTSDropProxy.cs

```diff
@@ -159,59 +122,74 @@ await Policy
          // Tried configureawait(false) and copyaync to each file (though not aparallelized) with no effect.
         public async Task Materialize(string localDestiantion)
         {
+            var uniquefiles = _files.GroupBy(file => file.Blob.Id).ToList();
```

Comment:
This is confusing. Do you mean uniqueBlobs? _files is already unique by definition...

Reply:
yeah this is files with the same content hash but different paths it's more than just the blobs but going back to that name.